runtime/util/fltmath.c

```diff
@@ -37,6 +37,28 @@
 #define J9_NO_DENORMAL_FLOAT_SUPPORT 1
 #endif
 
+I_32
+helperCDoubleAreDoublesEqual(jdouble a, jdouble b)
```

Comment:
you can pass the U_32 and U_64 directly and perform the reinterpret cast within the function directly